src/main/kotlin/org/cascadebot/cascadebot/commands/core/PageCommand.kt

```diff
@@ -14,7 +14,7 @@ class PageCommand : CoreCommand() {
     override fun command(): String = "page"
 
     override fun onCommand(sender: Member, context: CommandContext) {
-        if (context.args.isEmpty()) {
+        /*if (context.args.isEmpty()) {
```

Comment:
Is there a reason this has been commented out without a replacement in functionality?

Reply:
I forgot this existed. Will re-implement.

Reply:
Nvm we agreed to delete it lol